libs/langchain/pyproject.toml

```diff
@@ -316,6 +316,7 @@ cli = [
 # Please use new-line on formatting to make it easier to add new packages without
 # merge-conflicts
 extended_testing = [
+ "azureml-chat-endpoint",
```

Comment:
This causes the `tests/unit_tests/test_dependencies.py::test_required_dependencies` test to fail.
I don't know if there is a way to resolve it as contributor.
I added this dependency in order to skip the azureml_endpoint tests if the module is not installed as adviced [in this review](https://github.com/langchain-ai/langchain/pull/12377/files#r1374005711)